webots_ros2_core/webots_ros2_core/joint_state_publisher.py

```diff
@@ -57,10 +57,8 @@ def __init__(self, robot, jointPrefix, node):
 
     def publish(self):
         """Publish the 'joint_states' topic with up to date value."""
-        seconds = int(self.robot.getTime())
-        nanoseconds = int((self.robot.getTime() - seconds) * 1.0e+6)
         msg = JointState()
-        msg.header.stamp = Time(sec=seconds, nanosec=nanoseconds)
+        msg.header.stamp = Time(seconds=self.robot.getTime()).to_msg()
```

Comment:
It's probably ok, but are you sure that ``seconds`` handles floats?

Reply:
```
$ python3 -c 'from rclpy.time import Time; print(Time(seconds=0.32))'
Time(nanoseconds=320000000, clock_type=SYSTEM_TIME)
```
Yes, `Time` and `Duration` work nicely. Although I am disappointed there is no attribute `seconds` as I would like to do:
```
Time(nanoseconds=123).seconds
```
They may that, I checked their implementation and it should be straightforward.

Reply:
> They may that, I checked their implementation and it should be straightforward.

If it is really annoying, we might create classes that inherits from these classes and extend them, but this is probably a bit overkilled.